clang/lib/CodeGen/CGDynamicCheck.cpp

```diff
@@ -380,6 +380,17 @@ BasicBlock *CodeGenFunction::EmitDynamicCheckFailedBlock() {
   // Add a "failed block", which will be inserted at the end of CurFn
   BasicBlock *FailBlock = createBasicBlock("_Dynamic_check.failed", CurFn);
   Builder.SetInsertPoint(FailBlock);
+  if (getLangOpts().InjectVerifierCalls) {
```

Comment:
`*` and `&` should align with the variable name, not the type name.

`T* x ==> T *x`
`T& x ==> T &x`